.circleci/integration-tests/master_dag.py

```diff
@@ -169,6 +170,12 @@ def prepare_dag_dependency(task_info, execution_time):
         do_xcom_push=True,
     )
 
+    get_astronomer_providers_version = BashOperator(
+        task_id="get_astronomer_providers_version",
+        bash_command="airflow providers list -o json |  jq '.[] | select(.package_name == \"astronomer-providers\") | .version'",
```

Comment:
```suggestion
        bash_command="airflow providers get astronomer-providers -o json | jq '.[0].Version'",
```
I think this could work

Reply:
@pankajkoti tested the above  exiting command and it does return the correct [version](https://astronomer.slack.com/archives/C059006G6SG/p1687409797893279)